sunkit_magex/pfss/utils.py

```diff
@@ -36,6 +37,7 @@ def _earth_obs_coord_meta(obstime):
     return _observer_coord_meta(sunpy.coordinates.get_earth(obstime))
 
 
+@deprecated('1.0', message="This is now fixed within sunpy when you load a HMI file.")
 def fix_hmi_meta(hmi_map):
```

Comment:
So the only fix not in sunpy core is:

```
    # Fix observer coordinate
    if 'hglt_obs' not in hmi_map.meta:
        hmi_map.meta.update(_earth_obs_coord_meta(hmi_map.meta['date-obs']))
```
We can talk about adding it but that becomes an open question for the other SDO or earth based observatories.